### ISSUE TITLE
Incorrect gradients in chart trend emoji with shape reuse

### ISSUE THREAD

**Issue by drott** [googlefonts/nanoemoji#344]
In the following sequence of shape reuse and emoji `.svg` file arguments, the stock chart has an incorrect gradient tone in the lower part, or lower left corner. The gradient in the COLRv1 version that fills the stock chart grid appears too bright and almost fully white in the COLRv1 version.

![image](https://user-images.githubusercontent.com/1021784/134232666-9b106288-f4f5-4691-ad18-6d73079aa3d2.png)

This happens in the following reuse sequence, which includes the ticket emoji:

```
nanoemoji --gen_svg_font_diffs font-srcs/noto-emoji/svg/emoji_u1f4c8.svg font-srcs/noto-emoji/svg/emoji_u1f4c9.svg font-srcs/noto-emoji/svg/emoji_u1f39f.svg
```

If I remove the ticket emoji from this sequence (which is the last one in the sequence), i.e. 

```
nanoemoji --gen_svg_font_diffs font-srcs/noto-emoji/svg/emoji_u1f4c8.svg font-srcs/noto-emoji/svg/emoji_u1f4c9.svg 
```

the diff becomes smaller and the gradients appear more correct:

![image](https://user-images.githubusercontent.com/1021784/134232980-89c27447-aa80-415d-a552-9fd979ac8dca.png)

**Comment by drott** [comment]
These diff images were generated using nanoemoji main (with ClipBox fixes) + the not-yet-merged nanoemoji rotation fix. Or in other words, with this tree: https://github.com/drott/nanoemoji/tree/fixRotated

**Comment by anthrotype** [comment]
hm I cannot reproduce..

this is what I see on my end when I build the two fonts (with and without the ticket emoji) using the above commands and nanoemoji from the https://github.com/googlefonts/nanoemoji/tree/fix-rotated-clip-box branch (PR #342). My skia_colr's `colr_test` utility was built from [this branch on my fork](https://github.com/anthrotype/skia_colr/tree/update-skia) which includes the freetype rolled to head and the fix for composite/blending. I made a GIF so that you can see there are no visible changes (except for the PIL-generated pink diff in the middle)

![chart-emoji-diff](https://user-images.githubusercontent.com/6939968/134345486-71811b02-7681-459c-a176-b99b79255cc3.gif)

**Comment by anthrotype** [comment]
I was able to reproduce the bug. The difference is accounted by the different picosvg version, in my local env I was using 0.18.3,  whereas Dominik was using 0.18.2, which does not contain a [fix for Affine2D.inverse()](https://github.com/googlefonts/picosvg/pull/244)

In this commit I meant to bump picosvg version to 0.18.3 but I actually bumped it to 0.18.2 instead
https://github.com/googlefonts/nanoemoji/commit/2615273185c5c262055a3b6950c52239c9fd645b

I'll fix it by bumping it correctly this time and releasing a new nanoemoji
